fabric-biome-api-v1/src/main/java/net/fabricmc/fabric/impl/biome/modification/BiomeSelectionContextImpl.java

```diff
@@ -96,4 +100,15 @@ public boolean hasStructure(RegistryKey<ConfiguredStructureFeature<?, ?>> key) {
 		Registry<ConfiguredStructureFeature<?, ?>> registry = dynamicRegistries.get(Registry.CONFIGURED_STRUCTURE_FEATURE_KEY);
 		return registry.getKey(configuredStructure);
 	}
+
+	@Override
+	public boolean canGenerateIn(RegistryKey<DimensionOptions> dimensionKey) {
+		var dimension = levelProperties.getGeneratorOptions().getDimensions().get(dimensionKey);
```

Comment:
I think the convention is to not use `var`.

Reply:
Pssssssscht, no one noticed (and I honestly forgot)

Reply:
Fixed